test/sql.test.js

```diff
@@ -281,7 +281,7 @@ describe('sql connector', function() {
   });
 
   it('builds UPDATE', function() {
-    var sql = connector.buildUpdate('customer', {name: 'John'}, {vip: false});
+    var sql = connector._buildUpdate('customer', {name: 'John'}, {vip: false});
```

Comment:
Hmm, this change seems to indicate this patch is making a breaking change, as it removes a method that is a part of public API now.

Can we keep `buildUpdate` as an alias of `_buildUpdate`, perhaps with a deprecation notice?